### PR TITLE
Add Basic Documentation, Minor Docker and Postman Example Changes

### PR DESCRIPTION
**Pull request by anatolytsi**
This MR will add the basic documentation for the repository and its subfolders as well as minor changes to starting the python docker. 

The following is added:

* `.giattributes` files for running the project using Windows Docker
* Python backend `Dockerfile` order is edited to allow installing the dependancies before copying the rest of the files
* Modification to the postman example, i.e., minor refactoring and reordering
* Basic outline of the repository `README.md` with some information
* Basic folder `README.md` outlines of the Python backend
* Basic folder `README.md` outlines of the NodeJS backend
* Basic folder `README.md` outlines of the ParaView backend

### CODE REVIEW

**Comment by anatolytsi** [comment]
@FadySalama initial part is done, please give feedback on added parts and would be great to hear some ideas on unfilled parts.

**Comment by FadySalama** [comment]
This is really good @anatolytsi! Are you going to add something else or can I already merge this?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,168 @@
...
+
+1. [Web of Things Simulations using CFD](#web-of-things-simulations-using-cfd)
+2. [What can you do with WoT Phyngs Simulator](#what-can-you-do-with-wot-phyngs-simulator)
+3. [Getting started](#getting-started)
+- [Prerequisites](#prerequisites)
+- [Installing from sources](#installing-from-sources)
+- [Starting the Simulator](#starting-the-simulator)
+- [Setting up a Sample Simulation](#setting-up-a-sample-simulation)
+- [Control the Things](#control-the-things)
+- [Access the ParaView Server](#access-the-paraview-server)
```
Comment (FadySalama) [inline README.md]: Please make sure that indentation here is correct @anatolytsi

File: README.md [hunk 2]
```diff
@@ -1,2 +1,168 @@
...
+
+## Best Practices
+
+- Do not use too low of a mesh quality, the data might not be usable or the case would not be even created (depends on the objects complexity/amount).
+- Do not use too high of a mesh quality if you want to achieve the "realtime" and lack the resourceful server. The simulator is currently build on OpenFOAM version that utilizes CPU, not GPU. Hence, even the most powerful PC builds might not handle "realtime" in some "simple" cases.
+- Be realistic about the simulation geometric boundaries and do not put Phyngs outside the enclosed space (walls).
+- Windows, doors and other flat surfaces must lay exactly on some other surface, i.e., walls. When using custom STLs, it is even more complex to fulfill this.
+- Keep in mind that the using high quality STL and a coarse mesh would not bring much results. Sometimes it might give more problems while setting up.
+
+## Errors and Known Problems
```
Comment (FadySalama) [inline README.md:166]: Errors? What errors? :P
For real though, you should write about how the simulation collapses sometimes if the parameters aren't good etc. and that you can find this out if the simulation time is not progressing @anatolytsi

File: README.md [hunk 3]
```diff
@@ -1,2 +1,168 @@
...
+
+## Adding or Working on Issues
```
Comment (FadySalama) [inline README.md:180]: Similar to Wade, issues should have a naming convention: `[nodejs | paraview | python | misc] summarized issue description`. PRs solving issues should contain a comment saying exactly that. @anatolytsi

**Comment by anatolytsi** [comment]
Thank you @FadySalama! Surely, you can merge it. If where is anything else to add - I am at your service :)

**Comment by FadySalama** [comment]
Just fix the review points I listed above and I will merge it

**Comment by anatolytsi** [comment]
> Just fix the review points I listed above and I will merge it

I don’t see any review points, did you upload them?

**Comment by FadySalama** [comment]
I tagged you in them @anatolytsi. Can you see them now?

**Comment by anatolytsi** [comment]
> I tagged you in them @anatolytsi. Can you see them now?
Not at all… Your reviews are not displayed anywhere for me. Neither in Pull Request review, nor anywhere else.

**Review by FadySalama — CHANGES_REQUESTED** [review]
It seems that I didn't submit my review. Here it is!

File: README.md (hunk 2 above)
Comment (anatolytsi) [reply README.md:166]: No errors/problems found :D

**Comment by anatolytsi** [comment]
I have pushed the changes @FadySalama, please have a look and let me know if something more has to be changed

**Review by FadySalama — APPROVED** [review]
Really good, will merge

**Pull request merged**